processing/src/main/java/io/druid/query/filter/Filter.java

```diff
@@ -36,8 +42,15 @@
    *
    * @see Filter#estimateSelectivity(BitmapIndexSelector)
    */
-  ImmutableBitmap getBitmapIndex(BitmapIndexSelector selector);
+  default ImmutableBitmap getBitmapIndex(BitmapIndexSelector selector)
+  {
+    return getBitmapResult(selector, new DefaultBitmapResultFactory(selector.getBitmapFactory()));
+  }
 
+  default <T> T getBitmapResult(BitmapIndexSelector selector, BitmapResultFactory<T> bitmapResultFactory)
```

Comment:
@drcrallen probably you meant this comment on `getBitmapIndex()`, but even `getBitmapIndex()` is not deprecated _for usage_, for those who don't need BitmapResult thing. `getBitmapIndex()` is deprecated _for overriding_.